apps/glusterfs/operations_block_volume.go

```diff
@@ -647,6 +655,11 @@ func (bve *BlockVolumeExpandOperation) CleanDone() error {
 		} else {
 			logger.Info("Actual Size matches newly requested Size %v", bve.reclaim.actualSize)
 			bv.Info.Size = bve.newSize
+
+			if bve.reclaim.usableSize > 0 {
```

Comment:
Is this check required because heketi might be talking to an older gluster-block that does not have the usableSize field in info output? If so, can we please have that as a comment here?

Reply:
make sense, will this be fine?

```
 // Older versions of gluster-block might not have a field in the         
 // `# gluster-block info ...` output from which we calculate usableSize. 
 // In such cases reclaim.usableSize will be set to zero.                 
 if bve.reclaim.usableSize > 0 {                                          
   bv.Info.UsableSize = bve.reclaim.usableSize                            
   logger.Info("Usable Size is: %v", bv.Info.UsableSize)                  
 }                                                                        
```

Reply:
Yes, that looks good.

Reply:
cool